testsuite/tests/libratest/dev_modules/test_script.move

```diff
@@ -1,2 +1,4 @@
+script {
 fun main(_amount: u64, _addr: address) {
 }
+}
```

Comment:
It looks like you missed libratest/dev_modules/module.move. The "address" syntax there needs to be updated.